<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qjdoptions</class>
 <widget class="QDialog" name="qjdoptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkPID">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The task’s unique process ID, which periodically wraps, though never restarting at zero.
</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>PID</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="chkIOREAD">
       <property name="text">
        <string>IO_READ</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkCMD">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Process Name</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="chkIOWRITE">
       <property name="text">
        <string>IO_WRITE</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chkNice">
       <property name="toolTip">
        <string>The nice value of the task.  A negative nice value means higher pri-ority,  whereas a positive nice value means lower priority.  Zero in this field simply means priority will not be adjusted in determining a task`s dispatchability.
</string>
       </property>
       <property name="text">
        <string>Nice</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chkPCPU">
       <property name="toolTip">
        <string>The  task’s  share  of  the  elapsed  CPU time since the last screen update, expressed as a percentage of total CPU time.  In a true  SMP environment,  if  ’Irix  mode’  is Off, top will operate in ’Solaris mode’ where a task’s cpu usage will be divided by the  total  number of  CPUs.   You toggle ’Irix/Solaris’ modes with the ’I’ interactive command.
</string>
       </property>
       <property name="text">
        <string>%CPU</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="chkStartTime">
       <property name="toolTip">
        <string>Total CPU time the task has used since it started.  When ’Cumulative mode’  is  On,  each process is listed with the cpu time that it and its dead children has used.  You toggle ’Cumulative mode’ with  ’S’, which  is a command-line option and an interactive command.  See the ’S’ interactive command for additional  information  regarding  this mode.
</string>
       </property>
       <property name="text">
        <string>Running Time</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chkWCPU">
       <property name="text">
        <string>%WCPU</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="chkMem">
       <property name="text">
        <string>Memory</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="chkCMDline">
       <property name="toolTip">
        <string>Display the command line used to start a task or  the  name  of  the associated  program.   You toggle between command line and name with’c’, which is both a command-line option and an interactive command.
</string>
       </property>
       <property name="text">
        <string>COMMAND LINE</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="chkPMem">
       <property name="toolTip">
        <string>A task’s currently used share of available physical memory.
</string>
       </property>
       <property name="text">
        <string>Memory(%)</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="chkUID">
       <property name="toolTip">
        <string>The effective user ID of the task’s owner.
</string>
       </property>
       <property name="text">
        <string>UID</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="chkStackSize">
       <property name="text">
        <string>Stack size</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="chkUsrName">
       <property name="toolTip">
        <string>The effective user name of the task’s owner.
</string>
       </property>
       <property name="text">
        <string>User Name</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="chkWchan">
       <property name="toolTip">
        <string>Depending on the availability of the kernel link map (’System.map’), this field will show the name or the address of the kernel  function in which the task is currently sleeping.  Running tasks will display a dash (’-’) in this column.
</string>
       </property>
       <property name="text">
        <string>wchan</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="chkSleepAvg">
       <property name="text">
        <string>sleep average</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="chkWhichCPU">
       <property name="toolTip">
        <string>A  number representing the last used processor.  In a true SMP environment this will likely change frequently since the  kernel  intentionally  uses weak affinity.  Also, the very act of running top may break this weak affinity and cause more  processes  to  change  CPUs more often (because of the extra demand for cpu time).
</string>
       </property>
       <property name="text">
        <string>which cpu</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="chkStat">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>The status of the task which can be one of:
             ’D’ = uninterruptible sleep
             ’R’ = running
             ’S’ = sleeping
             ’T’ = traced or stopped
             ’Z’ = zombie
</string>
       </property>
       <property name="text">
        <string>Stat</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
